src/checks_windows.go

```diff
@@ -204,10 +216,52 @@ func packageInstalled(packageName string) (bool, error) {
 }
 
 func serviceUp(serviceName string) (bool, error) {
-	return commandOutput("(Get-Service -Name '"+serviceName+"').Status", "Running")
+	serviceStatus, err := getLocalServiceStatus(serviceName)
+	return serviceStatus.IsRunning, err
+}
+
+func serviceStatus(serviceName, wantedStatus, startupType string) (bool, error) {
+	status, err := getLocalServiceStatus(serviceName)
+	var boolWantedStatus bool
+	if err != nil {
+		return false, err
+	}
+	switch wantedStatus = strings.ToLower(wantedStatus); wantedStatus {
+	case "running":
+		boolWantedStatus = true
+	case "stopped":
+		boolWantedStatus = false
+	default:
+		errMessage := "Unknown status type found for " + serviceName
+		failPrint(errMessage)
```

Comment:
Can't you make this into one

Reply:
i made it into a var so i dont have to put it twice